### ISSUE TITLE
Some questions about NDR

### ISSUE THREAD

**Issue by bwtine** [yizhiwang96/deepvecfont#25]
This is such an amazing project！
How did you pre-train the NDR model? What are the labels for the training set and the test set?Can the NDR you provide be used to generate vector English characters for each style? Do I need to retrain NDR for each style English Fonts? 
How to train my own NDR for  icons or Chinese characteres？What are the labels for the training set and the test set?
I hope to get your reply, thanks!!!

**Comment by yizhiwang96** [comment]
Sorry for the late reply. NDR was designed to rasterize a drawing suquence (SVGs). If you learn a latent space from a datasets of SVGs, you can generate raster images of new fonts. The input is the drawing suquence processed by my code, the labels are the corresponding raster images. You can have a try!
